components/dashboard/src/teams/GitIntegrationsPage.tsx

```diff
@@ -4,48 +4,13 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { FunctionComponent } from "react";
-import { Redirect } from "react-router";
-import Header from "../components/Header";
-import { SpinnerLoader } from "../components/Loader";
-import { useCurrentOrg } from "../data/organizations/orgs-query";
 import { GitIntegrations } from "./git-integrations/GitIntegrations";
 import { OrgSettingsPage } from "./OrgSettingsPage";
 
-export default function GitAuth() {
+export default function GitAuthPage() {
     return (
-        <OrgSettingsPageWrapper>
+        <OrgSettingsPage title="Git Auth" subtitle="Configure Git Auth for GitLab or Github.">
             <GitIntegrations />
```

Comment:
**thought**: We can probably update the copy in the empty state for Git integrations, but we can make these in a separate PR.